web/person-roles/app/pages/manage-liquidators/[businessId].vue

```diff
@@ -0,0 +1,239 @@
+<script setup lang="ts">
+import type { FormSubmitEvent, FormErrorEvent } from '@nuxt/ui'
+import { z } from 'zod'
+
+const { t } = useI18n()
+const urlParams = useUrlSearchParams()
+const route = useRoute()
+const { setButtonControl } = useConnectButtonControl()
+const modal = useFilingModals()
+const liquidatorSchema = getLiquidatorsSchema()
+const liquidatorStore = useLiquidatorStore()
+const businessStore = useBusinessStore()
+const feeStore = useConnectFeeStore()
+const accountStore = useConnectAccountStore()
+
+// Dynamic i18n keys and breadcrumb based on intent to liquidate or manage liquidators
+// ToDo: include Liquidation report configuration when applicable
+const hasIntentToLiquidate = computed(() => route.path.includes('intent-to-liquidate'))
+const i18nKeys = computed(() => {
+  return hasIntentToLiquidate.value
+    ? {
+      h1: t('page.intentToLiquidate.h1'),
+      title: t('page.intentToLiquidate.title')
+    }
+    : {
+      h1: t('page.manageLiquidators.h1'),
+      title: t('page.manageLiquidators.title')
+    }
+})
+
+const { dashboardUrl, breadcrumbs } = useFilingNavigation(i18nKeys.value.h1)
+
+useHead({
+  title: i18nKeys.value.title
+})
+
+definePageMeta({
+  layout: 'connect-pay-tombstone-buttons',
+  middleware: [
+    // Check for login redirect
+    'connect-auth'
+  ],
+  alias: ['/manage-liquidators/:businessId/intent-to-liquidate'] // aliased path for intent to liquidate
```

Comment:
What do you think about: 
```
alias: ['/manage-liquidators/:businessId/:filingSubType']
```
and then I'm thinking below we can do
```
const filingSubType = route.params.filingSubType as FilingSubType 
```
and then pass that into the init / use it to determine what we show on the page later

Reply:
I like this too! although we will have to map the filingSubType from snake-case to camelCase, at some point.
As the routes coming in are snake and the enum definitions are kebab, pretty sure

Reply:
Gotcha, makes sense.

In that case, let's defer until the filing types/ sub types are sorted. 
Should be a small change, at that point.